frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -90,14 +92,20 @@ const MainLayoutContentContainer = styled('main')(({ theme }) => ({
 
 export const MainLayout = forwardRef<HTMLDivElement, IMainLayoutProps>(
     ({ children }, ref) => {
+        const newAdminUIEnabled = useUiFlag('adminNavUI');
         const { uiConfig } = useUiConfig();
+        const location = useLocation();
         const projectId = useOptionalPathParam('projectId');
         const { isChangeRequestConfiguredInAnyEnv } = useChangeRequestsEnabled(
             projectId || '',
         );
 
         const theme = useTheme();
         const isSmallScreen = useMediaQuery(theme.breakpoints.down('lg'));
+        const showOnlyAdminMenu =
```

Comment:
Wait, so how do you navigate out of the admin menu if you've landed there? If the other nav is gone, ... how do you get out?

Reply:
Right now - you'd have to edit URL or through command bar. It's turned off for devs for the time being, eventually there will be an admin menu that looks a little different (and also has a link back to dashboard)

Reply:
Got it. Seems weird to trap users there, but the link back should be good.